src/container/rpc.h

```diff
@@ -74,7 +74,8 @@
 	X(CONT_TGT_EPOCH_AGGREGATE, 0, &CQF_cont_tgt_epoch_aggregate,                              \
 	  ds_cont_tgt_epoch_aggregate_handler, &ds_cont_tgt_epoch_aggregate_co_ops)                \
 	X(CONT_TGT_SNAPSHOT_NOTIFY, 0, &CQF_cont_tgt_snapshot_notify,                              \
-	  ds_cont_tgt_snapshot_notify_handler, &ds_cont_tgt_snapshot_notify_co_ops)
+	  ds_cont_tgt_snapshot_notify_handler, &ds_cont_tgt_snapshot_notify_co_ops)                \
```

Comment:
I was wondering if that was necessary when the new RPC is server only, since all servers are expected to be the same version, at least until we implement the rolling upgrade.

Reply:
My understanding has been that if we had incremented DAOS_CONT_VERSION, then we might be able to catch mistakes that had unintentionally caused the engines to be different versions. Does that match what you have in mind, @kccain?

Reply:
I am more familiar with the client/engine negotiation aspect, which Kris pointed out is not in scope here. Hadn't thought about what would occur at runtime if engines of 2 different protocol versions tried to communicate, versus 2 engines with DAOS_CONT_VERSION=8 but actually having a different set of RPCs (maybe some type of -DER_UNREG error would occur if one engine tried to invoke CONT_PROP_SET_BYLABEL?).